pkg/kubelet/container/helpers.go

```diff
@@ -44,6 +44,7 @@ type RuntimeHelper interface {
 	GetClusterDNS(pod *api.Pod) (dnsServers []string, dnsSearches []string, err error)
 	GetPodDir(podUID types.UID) string
 	GeneratePodHostNameAndDomain(pod *api.Pod) (hostname string, hostDomain string, err error)
+	GetExtraSupplementalGroupsForPod(pod *api.Pod) []int64
```

Comment:
Wasn't sure where to put this, where would I make the utility function? (before, the volumemanager was passed to the runtime but I changed it because now volumemanager itself needs an initialized runtime)